irohad/torii/processor/impl/stub_query_processor.cpp

```diff
@@ -25,7 +25,9 @@ namespace iroha {
     using dao::QueryResponse;
     using dao::Client;
 
-    QueryProcessorStub::QueryProcessorStub() {
+    QueryProcessorStub::QueryProcessorStub(ametsuchi::WsvQuery &wsv,
+                                           ametsuchi::BlockQuery &block) :
```

Comment:
Yes, but in this case `QueryProcessorStub` will know about `AmetsuchiStub` methods (createTemporaryWsv and so on), which he doesn't need